website/docs/stitch-type-merging.md

```diff
@@ -514,6 +514,43 @@ import { SDC } from 'statsd-client';
 
 const statsd = new SDC({ ... });
 
+function createInstrumentedMergedTypeResolver(mergedTypeConfig) {
```

Comment:
Yeah, this is simpler... we should just make this the example. I've got some v7 notes to collect so will remove the redundant example with that.

Reply:
or if you're still making changes, pull the `Or:` example. Yours is simpler.

Reply:
i think we also may need another type to make clear that the selectionSet field from MergedTypeConfig is not used by createMergedTypeResolver...

Reply:
thoughts?

Reply:
Wait, actually – is your example going to work correctly? The subschema config passed through into the delegation methods won't have all the configuration used while building the resolver. Does that matter? If not, I'd say this could be as simple as mentioning which fields aren't used by the merged type resolver below.

Reply:
created the new type. tests say it will work. 👍 

the `resolve` method is built from `fieldName` and `args` or from `fieldName`, `argsFromKeys`, and `valuesFromResults`.

those settings are only necessary for building the `resolve` method -- and nothing else.